data/data/baremetal/bootstrap/main.tf

```diff
@@ -8,7 +8,38 @@ resource "libvirt_ignition" "bootstrap" {
   content = var.ignition
 }
 
-resource "libvirt_domain" "bootstrap" {
+resource "libvirt_domain" "bootstrap_single_interface" {
+  count = var.provisioning_bridge == "" ? 1 : 0
```

Comment:
This hack is used to have 2 definitions of the bootstrap VM, one with only an external interface, and one with both. We use the count property to select which bootstrap to use. I didn't find another solution.

TODO: I should add a comment to the terraform HCL here explaining this confusing monstrosity

Reply:
HCL terraform now supports dynamic blocks, why would we not use that? https://www.terraform.io/docs/configuration/expressions.html#dynamic-blocks